brigade.js

```diff
@@ -90,6 +90,11 @@ function buildLoggerLinux() {
   return buildImage("logger-linux");
 }
 
+// Build the worker
+function buildWorker() {
+  return buildImage("worker");
+}
+
```

Comment:
Let's remove this entire commit from this PR. Everything in here is already merged into the v2 branch.

It's not that it's going to cause a merge conflict-- it won't. It will just be easier to review if we can eliminate some noise.

Reply:
I don't always understand how/why GitHub sometimes shows me commit-specific comments in different contexts...

So to clarify, I am talking about commit bc3808da46021780a261c855eea5990e28711d42.

Reply:
Disregard this comment. It looks like this was all straightened out by a recent merge commit.